### ISSUE TITLE
Out of memory issue during training and testing

### ISSUE THREAD

**Issue by ZezhouCheng** [microsoft/human-pose-estimation.pytorch#141]
My env setting is Pytorch 0.4.0 + CUDA 9.0, I disabled the CUDNN batch norm. However, it seems that there is a GPU memory leak. Does anyone encounter this problem?

**Comment by ZezhouCheng** [comment]
Problem solved after switching to another server ...
